modules/110-istio/openapi/config-values.yaml

```diff
@@ -412,9 +431,71 @@ properties:
               format: int64
             value:
               type: string
+      replicasManagement:
+        description: |
+          Replication management settings and scaling of istiod.
+        type: object
+        default: {}
+        x-examples:
+        - mode: Standard
+        - mode: Static
+          static:
+            numReplicas: 3
+        x-doc-examples:
+        - mode: Standard
+        - mode: Static
+          static:
+            numReplicas: 3
+        - mode: HPA
+          hpa:
+            minReplicas: 2
+            maxReplicas: 5
+            cpuTargetAverageUtilization: 80
+        properties:
+          mode:
+            type: string
+            description: |
+              Replicas management mode:
+              - `Standard` — replicas management and scaling mode according to the global fault tolerance mode (the [highAvailability](../../deckhouse-configure-global.html#parameters-highavailability) parameter);
+              - `Static` — the mode, where the number of replicas is specified explicitly (the [static.numReplicas](#parameters-controlplane-replicasmanagement-static-numreplicas) parameter);
+              - `HPA` — the mode, where the number of replicas is calculated automatically using [HPA](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/) based on CPU usage. You can configure this mode by modifying parameters in the [hpa](#parameters-controlplane-replicasmanagement-hpa) parameter section.
+            enum: ['Standard', 'Static', 'HPA']
+            default: 'Standard'
+          static:
+            type: object
+            description: |
+              Options for replicas management for the `Static` mode.
+            required: ["numReplicas"]
+            properties:
+              numReplicas:
+                type: number
+                minimum: 1
+                description: |
+                  Desired number of replicas.
+          hpa:
+            type: object
+            description: |
+              Options for replicas management for the `HPA` mode.
+            required: ["minReplicas", "maxReplicas", "cpuTargetAverageUtilization"]
+            properties:
+              minReplicas:
+                type: number
+                minimum: 1
+                description: |
+                  The lower limit for the number of replicas to which the HPA can scale down.
+              maxReplicas:
+                type: number
+                minimum: 1
+                description: |
+                  The upper limit for the number of replicas to which the HPA can scale up. It cannot be less that `minReplicas`.
+              cpuTargetAverageUtilization:
```

Comment:
Is CPU utilization the only metric that users may want to use for HPA? What if we would like to add memory utilization in the future?

Reply:
```
mode: HPA
hpa:
  minReplicas: 2
  maxReplicas: 5
  metrics:
  - type: Classic
    resource: CPU
    target:
      averageUtilization: 70```